conf/config-default.yaml

```diff
@@ -202,6 +202,9 @@ etcd:
   # user: root                    # root username for etcd
   # password: 5tHkHhYkjr6cQY      # root password for etcd
   tls:
+    # cert: /path/to/cert         # path of certificate used by the etcd client
```

Comment:
Since we don't support mTLS actually in the runtime now, I think we should add a note here to tell users it and remove this note once we support it.

Reply:
I think we can support the whole stuff before the next release.

Reply:
Actually, it is just a breakdown of a large PR.

Reply:
OK.